src/migrations/20190503000724-create-salt.js

```diff
@@ -0,0 +1,21 @@
+export default {
+  up: (queryInterface, DataTypes) => {
+    return queryInterface.createTable('salts', {
+      id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        defaultValue: DataTypes.UUIDV4
+      },
+      value: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      createdAt: DataTypes.DATE,
```

Comment:
Not sure about what this means but I usually write the whole field's specification just to make it clear for the next person that will read it. I even include `allowNull: true` for nullable fields.

Reply:
Ok, I'll fix it. I followed Sequelize's documentation ([Models definition](http://docs.sequelizejs.com/manual/models-definition.html) Timestamps section)

Reply:
:+1: 

It's not an issue, it's just a suggestion. I'm just quoting Clean Code :joy: